features/hangouts/edit_hangout_url.feature

```diff
@@ -30,4 +30,12 @@ Feature: Manual Edit of Hangout URL
     And I manually set a hangout link for event "Repeat Scrum"
     Then "Repeat Scrum" shows live for that hangout link for the event duration
     And "Repeat Scrum" is not live the following day
-    And there should be exactly 3 hangouts
\ No newline at end of file
+    And there should be exactly 3 hangouts
+
+  Scenario: Event doesn't go live before Hangout URL is updated
+    Given the date is "2014 Feb 5th 6:00am"
+    And I manually set a hangout link for event "Repeat Scrum"
+    Given the date is "2014 Feb 5th 7:05am"
```

Comment:
To clarify, add "to a different URL" to line 40 and add the step (with possible refactoring to make it more DRY), then test that the scenario fails on the original/unfixed code base, then test that it passes with the fix to the models.